rawtodigi_kokkosview.h

```diff
@@ -0,0 +1,494 @@
+#ifndef rawtodigi_kokkos_h_
+#define rawtodigi_kokkos_h_
+
+#include <Kokkos_Macros.hpp>
+
+#include "kokkosConfig.h"
+#include "input.h"
+#include "output.h"
+
+namespace kokkos_serial {
```

Comment:
I'd like to have a comment for a future reader (if any)
```suggestion
// In principle the namespace should be KOKKOS_NAMESPACE, but for some reason that lead to weird compilation
// errros. To be investigated later.
namespace kokkos_serial {
```

Reply:
Added